web/source/kmwcallback.js

```diff
@@ -448,8 +447,8 @@
    */    
   keymanweb['KA'] = keymanweb.KA = function(n,ch,s)  // Keyboard_Any()
   {   
-    if(ch == '') return 0;
-    var Lix = s._kmwIndexOf(ch); //I3319
+    if(ch == '') return false;
+    var Lix = s._kmwIndexOf(ch,0); //I3319
```

Comment:
We should fix `String.prototype.kmwIndexOf` header parameters in `kmwstring.js` rather than adding a parameter here (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf)